.travis.yml

```diff
@@ -25,12 +25,12 @@ before_install:
   - go get github.com/pierrre/gotestcover
 
 install:
-  - "go get -d -t ./..."
+  - "echo No go get steps since we have vendored packages..."
```

Comment:
Note, I put an `echo` statement in here in case there was a default `install` target that did the `go get`, I wanted to make sure it is overridden. It also serves as a comment in case someone goes to add it, now they know not to.